manim/mobject/mobject.py

```diff
@@ -137,6 +136,9 @@ def add(self, *mobjects):
         return self
 
     def add_to_back(self, *mobjects):
+        """Adds (or moves) all *mobjects to the head of the self.submobjects list.
+        The head of the list is rendered first - i.e. at the back of the scene.
+        """
```

Comment:
Oneline descriptions should be a bit shorter, technical explanations can come afterwards.

```suggestion
        """Adds (or moves) all passed mobjects to the back of the scene.
        
        .. note::

            Technically, this is done by adding (or moving) the mobjects to
            the head of ``self.submobjects``. The head of this list is rendered
            first, which places the corresponding mobjects behind the
            subsequent list members.
        """
```